packages/cli/openapi-ir-to-fern/src/__test__/fixtures/x-fern-servers/openapi.yml

```diff
@@ -0,0 +1,25 @@
+openapi: 3.1.0
+info:
+  title: Test extension `x-fern-enum` for enums
+  version: 1.0.0
+paths: {}
+servers:
+  - url: https://api.example.com
+    x-fern-server-name: api
+    x-fern-environment: Production
+  - url: https://auth.example.com
+    x-fern-server-name: auth
+    x-fern-environment: Production
+  - url: https://api-sandbox.example.com
+    x-fern-server-name: api
+    x-fern-environment: Sandbox
+  - url: https://auth-sandbox.example.com
+    x-fern-server-name: auth
+    x-fern-environment: Sandbox
```

Comment:
Do these top level server urls also support `x-fern-server` ? 
```
servers:
  - url: https://api.example.com
    x-fern-server: 
      name: api # should be optional 
      environment: Production
```

Reply:
Yeah, will update the test def to include this one! Good catch